# Robot name goes here
snake:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  joint1Controller:
    type: effort_controllers/JointPositionController
    joint: joint1  
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 
    
  joint2Controller:
    type: effort_controllers/JointPositionController
    joint: joint2 
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 

  joint3Controller:
    type: effort_controllers/JointPositionController
    joint: joint3 
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 

  joint4Controller:
    type: effort_controllers/JointPositionController
    joint: joint4  
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 

  joint5Controller:
    type: effort_controllers/JointPositionController
    joint: joint5  
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 
    
  joint6Controller:
    type: effort_controllers/JointPositionController
    joint: joint6  
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 
    
  joint7Controller:
    type: effort_controllers/JointPositionController
    joint: joint7  
    pid : {p: 0.1 , i: 0.0 , d: 0.0} 
    